pages/join.js

```diff
@@ -21,6 +22,18 @@ function Join({ router }) {
     router.prefetch(profileUpdateURL);
   }, []);
 
+  function onChange() {
+    // I believe this is where the value gets sent to the API for the backend.
+    // value is the prop passed into this function.
+  }
+
+  const recaptchaReference = React.createRef();
```

Comment:
Since you're using hooks, you actually want to use `useRef`, like so:

```suggestion
  import { useRef } from 'react'
  // ...
  const recaptchaReference = useRef(null)
```

Reply:
ok, I think I am having trouble figuring out between class and function based components, that makes a lot more sense though. I have a tab open to pour over using hooks tonight on shift. 
And since we are just doing a crude attempt we can just feed the component empty functions? I was trying to figure out how to make it do something and I couldn't get it to click. I suppose I was way overthinking it. 
Thank you so much for the info! Im going to dig in tonight. Do I need to go in and change the code to how you wanted it? This is actually my first PR review!

Reply:
Empty functions are totally fine!

Reply:
Sorry this was a simplification!

You don't need the comment line and the import should be at the very top of the file.

Actually surprised the linter didn't error 🤔

Reply:
But we still do need the hook instead of `React.createRef()`